packages/web/package.json

```
@@ -10,33 +10,40 @@
   "license": "MIT",
   "author": "Motif",
   "type": "module",
-  "exports": "./dist/index.js",
+  "exports": {
+    ".": "./dist/index.js",
+    "./init": "./dist/init.js"
+  },
   "main": "dist/index.js",
   "files": [
-    "README.md",
-    "dist"
+    "dist/index.*",
+    "dist/init.*"
```

If I recall correctly, we also need these:

```suggestion
    "dist/init.*",
    "dist/global.*",
    "dist/types.*"
```

---

Hm good one, `globals.d.ts` is not being included in build output actually